relay-general/src/store/normalize.rs

```diff
@@ -823,6 +830,18 @@ impl Default for NormalizeProcessor<'_> {
     }
 }
 
+#[cfg(test)]
+fn get_empty_light_normalization_config<'a>() -> LightNormalizationConfig<'a> {
+    LightNormalizationConfig {
+        client_ip: None,
+        user_agent: None,
+        received_at: None,
+        max_secs_in_past: None,
+        max_secs_in_future: None,
+        breakdowns_config: None,
+    }
+}
```

Comment:
nit: You can put `#[derive(default)]` above `struct LightNormalizationConfig`, and then call `Default::default()` to get an empty config.

Reply:
Addressed in https://github.com/getsentry/relay/pull/1386.